### DISCUSSION TITLE
ttml显示未导入

### DISCUSSION THREAD

**Discussion by camilleKing2021** [amll-dev/applemusic-like-lyrics#242] [Q&A]
![1713756816810](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/49023410/423be419-4138-49d1-a23c-c27c2b1b6b85)
![1713756830254 (1)](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/49023410/d5ff935c-92b8-4676-b817-da8efe717664)
![1713756976537](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/49023410/7f666587-62ad-4412-8b64-0fdba0b970c3)
  "@applemusic-like-lyrics/core": "^0.1.3",
   "@applemusic-like-lyrics/vue": "^0.1.5"

**Comment by camilleKing2021** [comment]
@Steve-xmh 可以帮忙看一下吗

**Comment by Steve-xmh** [comment]
core 模块中的 ttml 模块是私有的，请换用其他模块，例如 lyric 模块
